javadoc generation for jans-orm is failing

**Describe the bug**
When generating javadoc for `jans-orm` module, it reports below mentioned errors and fails.

```
[ERROR] /home/dhaval/IdeaProjects/Janssen/jans/jans-orm/ldap/src/main/java/io/jans/orm/ldap/operation/LdapOperationService.java:85: error: exception not thrown: com.unboundid.ldap.sdk.LDAPException
[ERROR]      * @throws LDAPException
[ERROR]                ^
[ERROR] /home/dhaval/IdeaProjects/Janssen/jans/jans-orm/ldap/src/main/java/io/jans/orm/ldap/operation/impl/LdapConnectionProvider.java:106: error: reference not found
[ERROR]      * @throws EncryptionException
[ERROR]        ^
[ERROR] /home/dhaval/IdeaProjects/Janssen/jans/jans-orm/ldap/src/main/java/io/jans/orm/ldap/operation/impl/LdapConnectionProvider.java:107: error: reference not found
[ERROR]      * @throws EncryptionException
```
and

```
[ERROR] /home/dhaval/IdeaProjects/Janssen/jans/jans-orm/core/src/main/java/io/jans/orm/operation/auth/BCrypt.java:72: error: self-closing element not allowed
[ERROR]  * String stronger_salt = BCrypt.gensalt(12)<br />
[ERROR]                                             ^
```

**To Reproduce**
Steps to reproduce the behavior:
- go to janssen codebase root directory
- run `mvn -f jans-orm/pom.xml javadoc:javadoc`

**Expected behavior**
Build should be successful. currently it fails.

--- a/jans-orm/core/src/main/java/io/jans/orm/operation/auth/BCrypt.java
+++ b/jans-orm/core/src/main/java/io/jans/orm/operation/auth/BCrypt.java
@@ -51,25 +51,25 @@ import java.security.SecureRandom;
  * hashpw method with a random salt, like this:
  * <p>
  * <code>
- * String pw_hash = BCrypt.hashpw(plain_password, BCrypt.gensalt()); <br />
+ * String pw_hash = BCrypt.hashpw(plain_password, BCrypt.gensalt()); <br>
  * </code>
  * <p>
  * To check whether a plaintext password matches one that has been hashed
  * previously, use the checkpw method:
  * <p>
  * <code>
- * if (BCrypt.checkpw(candidate_password, stored_hash))<br />
- * &nbsp;&nbsp;&nbsp;&nbsp;System.out.println("It matches");<br />
- * else<br />
- * &nbsp;&nbsp;&nbsp;&nbsp;System.out.println("It does not match");<br />
+ * if (BCrypt.checkpw(candidate_password, stored_hash))<br>
+ * &nbsp;&nbsp;&nbsp;&nbsp;System.out.println("It matches");<br>
+ * else<br>
+ * &nbsp;&nbsp;&nbsp;&nbsp;System.out.println("It does not match");<br>
  * </code>
  * <p>
  * The gensalt() method takes an optional parameter (log_rounds) that determines
  * the computational complexity of the hashing:
  * <p>
  * <code>
- * String strong_salt = BCrypt.gensalt(10)<br />
- * String stronger_salt = BCrypt.gensalt(12)<br />
+ * String strong_salt = BCrypt.gensalt(10)<br>
+ * String stronger_salt = BCrypt.gensalt(12)<br>
  * </code>
  * <p>
  * The amount of work increases exponentially (2**log_rounds), so each increment

--- a/jans-orm/core/src/main/java/io/jans/orm/reflect/util/ReflectHelper.java
+++ b/jans-orm/core/src/main/java/io/jans/orm/reflect/util/ReflectHelper.java
@@ -149,7 +149,7 @@ public final class ReflectHelper {
 
     /**
      * Perform resolution of a class name.
-     * <p/>
+     * <br>
      * Here we first check the context classloader, if one, before delegating to
      * {@link Class#forName(String, boolean, ClassLoader)} using the caller's
      * classloader
@@ -176,7 +176,7 @@ public final class ReflectHelper {
 
     /**
      * Perform resolution of a class name.
-     * <p/>
+     * <br>
      * Same as {@link #classForName(String, Class)} except that here we delegate
      * to {@link Class#forName(String)} if the context classloader lookup is
      * unsuccessful.
@@ -200,7 +200,7 @@ public final class ReflectHelper {
 
     /**
      * Is this member publicly accessible.
-     * <p/>
+     * <br>
      * Short-hand for {@link #isPublic(Class, Member)} passing the member +
      * {@link Member#getDeclaringClass()}
      *

--- a/jans-orm/ldap/src/main/java/io/jans/orm/ldap/operation/LdapOperationService.java
+++ b/jans-orm/ldap/src/main/java/io/jans/orm/ldap/operation/LdapOperationService.java
@@ -80,9 +80,6 @@ public interface LdapOperationService extends PersistenceOperationService {
      * @return true if successfully added
      * @throws DuplicateEntryException
      * @throws ConnectionException
-     * @throws DuplicateEntryException
-     * @throws ConnectionException
-     * @throws LDAPException
      */
     boolean addEntry(String dn, Collection<Attribute> atts) throws DuplicateEntryException, ConnectionException;
 

--- a/jans-orm/ldap/src/main/java/io/jans/orm/ldap/operation/impl/LdapConnectionProvider.java
+++ b/jans-orm/ldap/src/main/java/io/jans/orm/ldap/operation/impl/LdapConnectionProvider.java
@@ -103,8 +103,6 @@ public class LdapConnectionProvider {
      * @throws NumberFormatException
      * @throws LDAPException
      * @throws GeneralSecurityException
-     * @throws EncryptionException
-     * @throws EncryptionException
      */
     protected void init(Properties props) throws NumberFormatException, LDAPException, GeneralSecurityException {
         String serverProp = props.getProperty("servers");

